ts/Grid/Pro/CellEditing/CellEditingComposition.ts

```diff
@@ -165,13 +165,25 @@ namespace CellEditingComposition {
             ](column, editModeOptions?.renderer || {});
         }
 
+        if (!CellRendererRegistry.types[staticRendererTypeName]) {
+            staticRendererTypeName = 'text';
+        }
+
         const staticRendererType = CellRendererRegistry.types[
             staticRendererTypeName
         ];
 
         let defRenderer = staticRendererType.defaultEditingRenderer;
```

Comment:
This might be nitpicky, but I don't like the variable name `defRenderer`, since my mind does not read `def` as `default`, but `definition`. Could we just use `defaultRenderer`? I don't see the benefit of short variable names

Reply:
the ultimately correct naming would be:

`staticRendererTypeName` -> `viewRendererTypeName`
`defRenderer` -> `editModeRendererTypeName`